tdvt/README.md

```diff
@@ -0,0 +1,72 @@
+# TDVT datasets
```

Comment:
In this tdvt Readme it would be helpful to add the LogicalQueryFormat we use and the command to run with unsigned taco files:
```
[Datasource]
LogicalQueryFormat = simple
CommandLineOverride = -DDisableVerifyConnectorPluginSignature=true
```

Reply:
And can we provide some context on what these are?